src/inmanta/protocol/methods_v2.py

```diff
@@ -1328,3 +1328,52 @@ def get_environment_metrics(
     :param end_interval: The end of the time window for which the metrics should be returned.
     :param nb_datapoints: The amount of datapoint that will be returned within the given time interval for each metric.
     """
+
+
+@typedmethod(path="/login", operation="POST", client_types=[ClientType.api], enforce_auth=False, api_version=2)
+def login(username: str, password: str) -> ReturnValue[model.LoginReturn]:
+    """Login a new user. When the login succeeds an authentication header is returned with the Bearer token set.
```

Comment:
```suggestion
    """Login a user. When the login succeeds an authentication header is returned with the Bearer token set.
```
Not a new user, right?

Reply:
indeed, a user that exists.